src/store/pageview.ts

```diff
@@ -3,10 +3,19 @@ import { PageView, StorageKeys, PageFeatureResult } from '../../types';
 
 class ViewStore {
   pageViews: PageView[];
+  maxAge: number;
+  storageSize: number;
 
-  constructor() {
+  constructor(maxAge?: number, storageSize?: number) {
```

Comment:
Why do we add this to the constructor if we do not plan to use it this way?

Reply:
Was being used in a former implementation (I'm doing TDD).
Will delete.

Reply:
I always remove unused code, this is just from a cognitive perspective to anyone else looking at it - when I saw the above I started searching for where you are creating an instance and passing in those values.

So yeah unless it is being used then I would remove, keeping the API simpler.

Reply:
I agree. Every piece of code should imply it's application somewhere.
I used it to set the defaults without hardcoding it on the constructor, tho.
Do you rather have it directly on the constructor and these inputs deleted?
I guess it would be simpler.